src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-mount-details/cephfs-mount-details.component.ts

```diff
@@ -11,25 +14,98 @@ import { MountData } from '~/app/shared/models/cephfs.model';
 export class CephfsMountDetailsComponent extends BaseModal implements OnInit {
   @ViewChild('mountDetailsTpl', { static: true })
   mountDetailsTpl: any;
+  mountDetail!: Record<string, any>;
   onCancel?: Function;
   private MOUNT_DIRECTORY = '<MOUNT_DIRECTORY>';
+  filteredUsers: CephUser[] = [];
+  selectedUser: string = 'client.admin';
+  updatedUser: string;
+  publicaddr: string;
+  secret: string;
+  userForm: CdFormGroup;
+  mount!: string;
+  cephfsmount!: string;
+  fuse!: string;
+  nfs!: string;
+  extractedKey!: string;
+  uniqueIp: string[];
```

Comment:
are all these variables used? i saw some unused. can you recheck and remove if something is not used.